src/Swarm/Web.hs

```diff
@@ -68,19 +79,21 @@ webMain baton port gsRef = catch (Warp.runSettings settings app) handleErr
 defaultPort :: Warp.Port
 defaultPort = 5357
 
+data StateRefs = StateRefs (IORef GameState) (IORef UIState)
```

Comment:
What's the benefit of having two separate IORefs?  Why not just have a single `IORef  AppState`?

Reply:
Good point, will do.